sample_analyze_read.py

```diff
@@ -0,0 +1,192 @@
+       
+    ------Set up your programming environment------
+    At a command prompt,run the following code to install the Azure AI Document Intelligence client library for Python with pip:
+    pip install azure-ai-documentintelligence --pre
+    
+    ------Create your Python application------
+    1) Create a new Python file called "sample_analyze_layout.py" in an editor or IDE.
+    2) Open the "sample_analyze_layout.py" file and insert the provided code sample into your application.
+    3) At a command prompt, use the following code to run the Python code: 
+       python sample_analyze_read.py
+"""
+
+import os
+
+
+def get_words(page, line):
+    result = []
+    for word in page.words:
+        if _in_span(word, line.spans):
+            result.append(word)
+    return result
+
+# To learn the detailed concept of "span" in the following codes, visit: https://aka.ms/spans 
+def _in_span(word, spans):
+    for span in spans:
+        if word.span.offset >= span.offset and (word.span.offset + word.span.length) <= (span.offset + span.length):
+            return True
+    return False
+
+
+def analyze_read():
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.documentintelligence import DocumentIntelligenceClient
+    from azure.ai.documentintelligence.models import DocumentAnalysisFeature, AnalyzeResult, AnalyzeDocumentRequest
+
+    # For how to obtain the endpoint and key, please see PREREQUISITES above.
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+    document_intelligence_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+
+    # Analyze a document at a URL:
+    formUrl = "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/rest-api/read.png"
+    # Replace with your actual formUrl:
+    # If you use the URL of a public website, to find more URLs, please visit: https://aka.ms/more-URLs 
+    # If you analyze a document in Blob Storage, you need to generate Public SAS URL, please visit: https://aka.ms/create-sas-tokens
+    poller = document_intelligence_client.begin_analyze_document(
+        "prebuilt-read",
+        AnalyzeDocumentRequest(url_source=formUrl)
+    )       
+    # # If analyzing a local document, remove the comment markers (#) at the beginning of these 11 lines.
+    # # Delete or comment out the part of "Analyze a document at a URL" above.
+    # # Replace <path to your sample file>  with your actual file path.
+    # path_to_sample_document = "<path to your sample file>"
+    # with open(path_to_sample_document, "rb") as f:
+    #     poller = document_intelligence_client.begin_analyze_document(
+    #         "prebuilt-read",
+    #         analyze_request=f,
+    #         features=[DocumentAnalysisFeature.STYLE_FONT],
```

Comment:
There's addon feature turned on here, optional, you need to either delete it or make some comments on it for addon capability；）

Reply:
Is my understanding correct?
![image](https://github.com/luzhang06/sample/assets/161273051/e887ccab-2a16-4616-8a0f-58d23dc1280e)